compiler/damlc/daml-prim-src/GHC/Err.daml

```diff
@@ -14,10 +15,28 @@ module GHC.Err
 import GHC.Integer.Type() -- required so Integer optimisation works
 import GHC.Types
 
+#ifdef DAML_EXCEPTIONS
+
+import DA.Internal.Exception.Types
+
+-- | `error` stops execution and displays the given error message.
+--
+-- If called within a transaction, it will abort the current transaction.
+-- Outside of a transaction (scenarios, Daml Script or Daml Triggers)
+-- it will stop the whole scenario/script/trigger.
+--
+-- Throws a `GeneralError` exception.
+error : Text -> a
+error msg = primitive @"EThrow" (GeneralError msg)
```

Comment:
I assume there’s a bootstrapping/dependency problem that forces us to use `primitive` instead of just calling `throw`?

Reply:
Yes, throw is defined in daml-stdlib